File: spring-boot-react-example/src/main/java/com/rogerlester/app/tech/Tech.java
package com.rogerlester.app.tech;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name = "tech_tbl")
public class Tech {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String name;
    private String description;
    private String layer;
    private String language;

    public Tech() {}

    public Tech(String name, String description, String layer, String language) {
        this.name = name;
        this.description = description;
        this.layer = layer;
        this.language = language;
    }
}

File: spring-boot-react-example/src/main/js/components/tech/Tech.js
import React, { Component } from 'react'

export default class Tech extends Component {
    render() {
        const {name, description, layer, language} = this.props.tech
        return (
			<tr>
				<td>{name}</td>
				<td>{description}</td>
				<td>{layer}</td>
				<td>{language}</td>
			</tr>
        )
    }
}
